### ISSUE TITLE
Epoch number has inconsistent display in ModelCheckpoint

### ISSUE THREAD

**Issue by JiangYize** [PyTorchLightning/pytorch-lightning#2667]
## 📚 Documentation

Epoch 1: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████▊| 996/997 [02:32<00:00,  6.51it/s, loss=0.655, v_num=59]
Epoch 00000: val_loss reached 0.67374 (best 0.67374), saving model to ../checkpoints/epoch=0.ckpt as top 1██████████████████████████████████████████████████████████████████████████████████▍  | 137/139 [00:17<00:00,  9.99it/s]

**Comment by williamFalcon** [comment]
@Borda i think the tqdm should also just start at 0...

**Comment by Borda** [comment]
epochs on display starts from 1 compare to all internal indexing starts from 0
https://github.com/PyTorchLightning/pytorch-lightning/blob/1a54ed6ad9f1faf8ac58bbded4b71e4dd18246d6/pytorch_lightning/callbacks/progress.py#L328

**Comment by williamFalcon** [comment]
fixed
